### PR TITLE
162 create usermanual

### PR DESCRIPTION
**Pull request by jessicakjellin**
Updated README.md to match our user manual and added additional info

### CODE REVIEW

**Review by edvgi966 — APPROVED** [review]

**Review by grufe — CHANGES_REQUESTED** [review]
Look very nice!

Should also add that to analyze SBOMS you have to put the JSON of the SBOM in the
/sboms directory.

**Comment by grufe** [comment]
We also somehow have to change the -v flag. Now it has two uses

**Comment by jessicakjellin** [comment]
adding some finishing touches

**Review by KarlssonPaGolvet — APPROVED** [review]

**Review by grufe — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,2 +1,140 @@
...
+To install OSSQA, clone this repository.
+```
+git clone https://github.com/OSSQA-PUM/OSSQA.git
+```
+
+### Inserting SBOMs
+To analyze your SBOMs, the SBOM in CycloneDX-format needs to be added to the `/sboms` directory.
+
+### Inserting GitHub Access Token
+Insert your GitHub Access Tokej the `.env` file.
```
Comment (grufe) [inline README.md:21]: Tokej

**Review by grufe — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (grufe) [inline README.md:21]: Insert your GitHub Access Tokej the `.env` file.

Misspelling at Tokej

**Review by grufe — CHANGES_REQUESTED** [review]

**Pull request merged**